_episodes/06-style-guide.md

```diff
@@ -3,13 +3,24 @@ title: "Style Guide"
 teaching: 10
 exercises: 0
 questions:
+- "What tone and style should be used for Carpentries lessons?"
 - "How are keyboard key combinations written?"
+- "How are short spans of code written?"
```

Comment:
There is a brief mention of "short spans of code" in episode two (I think).  I wonder if we should consolidate the episodes a little bit... Not in this PR, however.